src/middleware/ErrorHandling.js

```diff
@@ -0,0 +1,37 @@
+// const UserServiceError = require('../exceptions/UserServiceError');
+
+/**
+ * Handle an error that can occur during the route handling
+ * @param {Object} err
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} next
+ * @return {Object}
+ */
+const handleError = (err, req, res, next) => {
+  console.error({
```

Comment:
@dpapukchiev you had something like `req.log.error(...)` here - for me it gave an error - I didn't really look into what was the issue, but maybe you know on top of your head?